openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -139,6 +145,16 @@ jQuery(function () {
             .then((module) => module.initPatronMetadata());
     }
 
+    if ($('#cboxPrevious').length) {
+        $('#cboxPrevious').remove();
+    }
+    if ($('#cboxNext').length) {
+        $('#cboxNext').remove();
+    }
+    if ($('#cboxSlideshow').length) {
+        $('#cboxSlideshow').remove();
+    }
+
```

Comment:
I'm not really comfortable with removing these buttons.  These are added by Colorbox, the library that we use for our modals.  My worry is that removing these may inadvertently break something in one of our modals.

Reply:
@jamesachamp 
Aria label attribute could be added to the buttons instead. This wouldn't require buttons to be removed.

Reply:
And should I include the attribute aria hidden as well?

Reply:
I think that the changes you've made are good. Thanks so much!